src/controllers/user/Controller.ts

```diff
@@ -69,22 +69,24 @@ class UserController {
             code: 200
         });
       }
-    catch (err) {
+    catch (err)  {
         res.send({
             error: 'User Not Found for update',
             code: 404
         });
     }
 }
 
-    login(req: Request , res: Response , next: NextFunction) {
-      try { const { email, password } = req.body;
-      userModel.findOne ( { email }, (err, result) => {
+  public async login(req: IRequest , res: Response , next: NextFunction) {
+      try {
+        const { email, password } = req.body;
+     await userModel.findOne ( { email }, (err, result) => {
           if ( result ) {
-              if ( password === result.password ) {
+              if ( password === config.Password ) {
```

Comment:
Use bcrypt to compare hashed passwords (As the user will send the password in plain text, but in DB you have the password in hashed format). Use bcrypt.compareSync() or bcrypt.compare() to match password.